bind_uploader.go

```diff
@@ -111,15 +111,18 @@ func (bu *bindUploader) createStageIfNeeded() error {
 		return err
 	}
 	if !data.Success {
-		code, err := strconv.Atoi(data.Code)
-		if err != nil {
-			return err
+		code, atoiErr := strconv.Atoi(data.Code)
+		if atoiErr != nil {
+			return atoiErr
 		}
 		return (&SnowflakeError{
 			Number:   code,
 			SQLState: data.Data.SQLState,
-			Message:  err.Error(),
-			QueryID:  data.Data.QueryID,
+			// This err will always be nil because we are returning after the exec if err is not nil
```

Comment:
By looking at the code it seems that indeed passing err here is incorrect.  But there is a `data.Message` field which I believe is best here.

Reply:
ok cool that answers my question in the comment 😄